### PR TITLE
First spike at getting tests to pass on dbt 0.20.b1

### LINKED ISSUE #59: Update expectations to at least pass integrations tests for dbt==0.20.0-b1
We need to rewrite expectations to return failing rows, not counts

### PR DESCRIPTION
**Pull request by clausherther**
Closes #59

### CODE REVIEW

File: macros/schema_tests/table_shape/expect_table_column_count_to_equal.sql [hunk 1]
```diff
@@ -1,6 +1,8 @@
 {%- macro test_expect_table_column_count_to_equal(model, value) -%}
 {%- if execute -%}
 {%- set number_actual_columns = (adapter.get_columns_in_relation(model) | length) -%}
-select {{ (number_actual_columns - value) | abs }}
+select 1
+from (select 1) a
+where {{ number_actual_columns }} != {{ value }}
```
Comment (jtcohen6) [inline macros/schema_tests/table_shape/expect_table_column_count_to_equal.sql:6]: This one, and the ones below it, feel tricky—especially to find the right boundary between Jinja calculation and SQL calculation. I could see wanting to return a row that contains two columns, `number_actual_columns` and `value`, and then set `fail_calc` to be their difference.
```sql
{{ config(fail_calc = "abs(number_actual_columns - value)") }}
select {{ number_actual_columns }} as number_actual_columns, {{ value }} as value
```
That way the debug query / stored-in-database table is just slightly more informative. What do you think?

**Review by jtcohen6 — COMMENTED** [review]
Glad this refactor wasn't too tricky! I'm going to open one other issue, which relates to `dispatch` changes in 0.20.0

File: macros/schema_tests/table_shape/expect_table_column_count_to_equal.sql (hunk 1 above)
Comment (clausherther) [reply macros/schema_tests/table_shape/expect_table_column_count_to_equal.sql:6]: Thanks @jtcohen6! I took some of these suggestions into a new branch!

**Comment by clausherther** [comment]
Closing in favor of #82

**Pull request closed without merging**